backend/.env.dist

```diff
@@ -63,5 +63,12 @@ EVENT_PROTOCOL_DISABLED=false
 
 # DHT
 # if you set this value, the DHT hyperswarm will start to announce and listen
-# on an hash created from this tpoic
+# on an hash created from this topic
 # DHT_TOPIC=GRADIDO_HUB
+# Define the following federation properties to individualize this community instance
+# for federation per hyperswarm
+# FEDERATE_COMMUNITY_NAME=Gradido Community Development
+# FEDERATE_COMMUNITY_URL==http://localhost/
+# FEDERATE_COMMUNITY_PORT=4000
+# FEDERATE_COMMUNITY_UUID=14466b15-4631-4b1b-9110-dd6694356d9c
+# FEDERATE_KEY_SECRET=a51ef8ac7ef1abf162fb7a65261acd7a
```

Comment:
is this the secret key or the key seed?

while I agree on `FEDERATE` on the above, this one is part of the DHT hyperswarm right? So it should be `DHT_...`

Reply:
it is the secret used as seed while creating a  key.

I would prever to rename the DHT_TOPIC to FEDERATE_DHT_TOPIC, because the hyperswarm is the first part of the whole federation process.

Reply:
That would be ok with me.

But please name the key seed as such, its a better technical description

Reply:
Done